release-notes/opendistro-for-elasticsearch-anomaly-detection-kibana-plugin.release-notes-1.10.1.0.md

```diff
@@ -0,0 +1,52 @@
+## Version 1.10.1.0 Release Notes
+
+Compatible with Kibana 7.9.1
+
+### Features
+
+* Add sample detectors and indices ([#272](https://github.com/opendistro-for-elasticsearch/anomaly-detection-kibana-plugin/pull/308272))
```

Comment:
not sure what happened to the links, but they're all prefixed w/ 308. I just ran the script to see if there was a bug, it seems to work fine for me. Are you saving the raw draft release notes?

Reply:
Yeah I ran with with the script. I updated it with a clean copy, it looks good now. Thanks for the catching it.